frontend/src/pages/team/create.vue

```diff
@@ -11,12 +11,12 @@
     <ff-page>
         <ff-loading v-if="redirecting" message="Redirecting to Stripe..." />
         <ff-loading v-else-if="loading" message="Creating Team..." />
-        <div v-else class="max-w-2xl m-auto">
+        <div v-else class="m-auto">
             <form class="space-y-6">
                 <FormHeading>Create a new team</FormHeading>
                 <div class="mb-8 text-sm text-gray-500">Teams are how you organize who collaborates on your projects.</div>
                 <!-- TeamType Type -->
-                <div class="flex flex-wrap gap-1 items-stretch">
+                <div class="grid grid-1">
```

Comment:
```suggestion
                <div class="grid">
```

Not sure `grid-1` does anything? It's not a tailwind class that I can find, and it renders the same in my browser if I remove it

Reply:
Was part of the other pr. Will apply and tidy up the other instance to match

Reply:
Have pushed a commit that fixes this in both this form and the upgrade form to match.